.ci/common/test.sh

```diff
@@ -73,6 +73,11 @@ run_oldtests() {
   valgrind_check "${LOG_DIR}"
 }
 
+run_single_includes_tests() {
+  ${MAKE_CMD} -C "${BULID_DIR}" check-single-includes
+}
+
+
```

Comment:
Sorry, but if I could continue to try activate check-single-includes test on TravisCI, this change should be kept.